src/widgets/PageInfo.js

```diff
@@ -104,7 +110,7 @@ function renderInfoComponent(isEditingDisabled, columnIndex, rowData, rowIndex)
       break;
   }
   // If null or empty string, use single space to avoid squishing row
-  let infoString = rowData.info && String(rowData.info);
+  let infoString = (rowData.info || rowData.info === 0) && String(rowData.info);
```

Comment:
!0 is true, so yes falsy.

I understand now, because 0 is falsy this becomes a weird edge case.